src/class-savvii-dashboard.php

```diff
@@ -162,13 +154,60 @@ function warpdrive_dashboard() {
                     <!-- /Caching defaults -->
             <?php endif; ?>
                     <!-- Read server logs -->
-                    <div class="postbox" style="min-height: 150px;">
+                    <div class="postbox">
                         <h2 class="hndle">Read server logs</h3>
                         <div class="inside"><?= is_super_admin() ? 'Please use the Savvii top menu for reading logs.' : '
                             \'Read server logs\' shows data for all subsites. Because of this, only users with the \'Super Admin\' role are able to read the server logs.'; ?>
                         </div>
                     </div>
                     <!-- /Read server logs -->
+                    <!-- Statistics -->
+                    <div class="postbox">
+                        <h2 class="hndle">Statistics</h2>
+                        <div class="inside">
+                            <div class="pct50">
+                            <?php
+                            $logreader = new ReadLogs();
+                            $errors_502 = $logreader->find_match_in_log( 'access', '/HTTP\/(?:1\.1|2)\"\s+502/i');
+                            $errors_504 = $logreader->find_match_in_log( 'access', '/HTTP\/(?:1\.1|2)\"\s+504/i');
+                            ?>
+                            <p>WarpDrive detected
+                                <?php
+                                echo ' '.count( $errors_502 ).' ';
+                                ?>
+                            recent 502 (Bad Gateway) error(s)</p>
+                            <p>WarpDrive detected <?php echo count( $errors_504 ); ?> recent 504 (Gateway Timeout) error(s)</p>
+                        </div>
+                        <div class="pct50">
+                            <?php
+                            $database = new Database();
+                            $results = $database->get_wp_table_sizes(10);
```

Comment:
This looks like duplicate code from the database size page, is it an idea to gather the rendering of the data in a function used by both pages?